webui/react/src/components/NavigationTabbar.tsx

```diff
@@ -111,6 +111,11 @@ const NavigationTabbar: React.FC = () => {
       ),
   );
 
+  const openWorkspaceCreateModal = () => {
+    clusterStore.fetchKubernetesResourceManagers();
+    WorkspaceCreateModal.open();
+  };
```

Comment:
Same maybe use of `useCallback` here and throughout.

Reply:
Seems like this is a non-issue per [this comment](https://github.com/determined-ai/determined/pull/9735#discussion_r1695364766). Resolving.